src/components/Utilities/ColorMapping.js

```diff
@@ -1,4 +1,4 @@
-import tinycolor from 'tinycolor2'
+// import tinycolor from 'tinycolor2'
```

Comment:
If we aren't planning on using tinycolor2 in the future just go ahead and remove the comments (and don't forget the package.json)

Reply:
It was commented out to potentially be used later for theming purposes. But sure.